src/frame-validator.cpp

```diff
@@ -66,10 +66,13 @@ namespace librealsense
         //after the validation all streams will be passeded to user callback directly
         if (stream->get_stream_type() != RS2_STREAM_INFRARED)
             return false;
+        // TODO review the above statement and check with PLM
 
+#if 0  // TODO WHY?
```

Comment:
@ev-mp 
See RS5-7654:
This solves the issue of multiple frame drops at the beginning of the stream.
I'll remove the few lines, though the ticket isn't closed because of lack of direction.

Reply:
Pushed